app/views/boats/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="container">
```

Comment:
I have implemented container for all views by adding this in the application.html.erb

<body>
  <p class="notice"><%= notice %></p>
  <p class="alert"><%= alert %></p>
  <div class = "container">
    <%= yield %>
  </div>
  </body>